src/rust/engine/src/python.rs

```diff
@@ -273,8 +273,8 @@ impl Key {
 pub struct Value(Arc<PyObject>);
 
 // NB: The size of objects held by a Graph is tracked independently, so we assert that each Value
-// is only as large as its pointers.
-known_deep_size!(8 * 3; Value);
+// is only as large as its pointer.
+known_deep_size!(8; Value);
```

Comment:
Instead of assuming the size is 8, maybe use `std::mem::size_of<* cosnt ()>` or some other pointer type?

Reply:
Mm, yea: good idea. I think that I wasn't sure whether it would work with the macro.